### PR TITLE
Format partitions for Stretch with XFS ftype support

### PR DESCRIPTION
**Pull request by kofrezo**

### CODE REVIEW

**Comment by TuxPowered42** [comment]
retest this please

File: igvm/hypervisor.py [hunk 1]
```diff
@@ -730,8 +731,11 @@ def umount_temp(self, device_or_path):
     def remove_temp(self, mount_path):
         self.run('rmdir {0}'.format(mount_path))
 
-    def format_storage(self, device):
-        self.run('mkfs.xfs -f {}'.format(device))
+    def format_storage(self, device, os):
+        if os == 'stretch':
```
Comment (TuxPowered42) [inline igvm/hypervisor.py]: This only verifies OS of VM but it should check HV too.

**Comment by kofrezo** [comment]
We don't have it enabled on all VMs as showed to you @HerrSpace so I think it is still a good idea to enable it manually if it is aleady enabled because of CRC it does no harm.

**Comment by HerrSpace** [comment]
@kofrezo I think we must find out on which HV the VM that didn't have it was build on. My guess is that HVs XFS just doesn't have it's feature and we rather must upgrade that. Maybe the VM was build on a Jessie HV?

File: igvm/hypervisor.py [hunk 2]
```diff
@@ -730,8 +731,11 @@ def umount_temp(self, device_or_path):
     def remove_temp(self, mount_path):
         self.run('rmdir {0}'.format(mount_path))
 
-    def format_storage(self, device):
-        self.run('mkfs.xfs -f {}'.format(device))
+    def format_storage(self, device, os):
```
Comment (TuxPowered42) [inline igvm/hypervisor.py]: Wouldn't it be cleaner to give only True/False here for ftype parameter and determine it above?

**Comment by TuxPowered42** [comment]
retest this please

**Comment by TuxPowered42** [comment]
retest this pleae

**Comment by TuxPowered42** [comment]
retest this please

**Pull request closed without merging**